src/kOS/Screen/TermWindow.cs

```diff
@@ -38,23 +38,23 @@ public class TermWindow : KOSManagedWindow , ITermWindow
         
         private bool collapseFastBeepsToOneBeep = false; // This is a setting we might want to fiddle with depending on opinion.
 
-        private KeyBinding rememberThrottleCutoffKey;
-        private KeyBinding rememberThrottleFullKey;
-
```

Comment:
These two lines were unused fields causing a warning message.  (They stopped being used once we stopped needing to have an ugly ad-hoc fix for locking out the 'Z' and 'X' keys.)

Deleting them was not related to this PR's purpose, but I removed them since I was in this file anyway and I noticed the warnings.